src/apps/profiles/forms.py

```diff
@@ -0,0 +1,19 @@
+from django import forms
+from django.contrib.auth.forms import UserCreationForm
+# from django.contrib.auth.models import User
+from .models import User
+
+
+class SignUpForm(UserCreationForm):
```

Comment:
Pull this, we shouldn't need it I don't think??? Also, just do HTML instead of Form templates, so we don't have to edit placeholders in Python.